package.nls.de.json

```diff
@@ -1,4 +1,5 @@
 {
+  "command.activateIcons.title": "",
```

Comment:
In fact, Duden(you may know that) also says that `Icon` is German aswell. http://www.duden.de/rechtschreibung/Icon

Dateisymboldesign is a neologism...